java/fury-core/src/main/java/org/apache/fury/config/FuryBuilder.java

```diff
@@ -336,6 +345,11 @@ public FuryBuilder withName(String name) {
     return this;
   }
 
+  public FuryBuilder withFieldMismatchCallback(FieldMismatchCallback callback) {
+    this.fieldMismatchCallback = callback;
```

Comment:
Maybe it can be confusing that the callback can be overridden, not added on top of already registered ones?